zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -39,14 +45,53 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
     version = BuildConfig.KOTLIN_PLUGIN_VERSION,
   )
 
+  override fun apply(project: Project) {
+    super.apply(project)
+
+    val extension = project.extensions.findByType(KotlinMultiplatformExtension::class.java)
+      ?: return
+
+    extension.targets.withType(KotlinJsIrTarget::class.java).all { kotlinTarget ->
+      kotlinTarget.binaries.withType(JsIrBinary::class.java).all { kotlinBinary ->
+        registerCompileZiplineTask(project, kotlinBinary)
+      }
+    }
+
+    project.tasks.withType(Delete::class.java).getByName("clean") { clean ->
```

Comment:
```suggestion
    project.tasks.named("clean", Delete::class.java).configure { clean ->
```
Existing version eagerly configures the task